wdl2cwl/tests/cwl_files/literal_struct.cwl

```diff
@@ -0,0 +1,40 @@
+cwlVersion: v1.2
+id: literal_struct_test
+class: CommandLineTool
+requirements:
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: script.bash
+        entry: |4
+
+            echo $({ "one": "four", "two": "five" }.one) $({ "one": "four", "two": "five" }.two) $(inputs.six.one) $(inputs.six.two)
```

Comment:
Maybe we could have parsed this down to something briefer, but I'm not so motivated at the moment as it works